backend/src/middlewares/upload.middleware.ts

```diff
@@ -0,0 +1,12 @@
+import multer from 'fastify-multer';
+
+const storage = multer.memoryStorage();
+
+const upload = multer({
+  storage,
+  limits: {
+    fileSize: 10000000,
```

Comment:
It's better to set constants like that in this way
```suggestion
    fileSize: 10 * 1024 * 1024, // 10 MB
```